src/rules/ResourceQuoteStyle.js

```diff
@@ -52,6 +52,14 @@ class ResourceQuoteStyle extends ResourceRule {
             this.localeOnly = true;
         }
         this.link = "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-quote-style.md";
+        if (!this.skipLocales) {
+            this.skipLocales = new Set();
+        }
+        
+        [
+            "sv-SE", // According to the MS Style guidelines, quotes are usually not required in Swedish when the source English text contains quotes
+            "it-IT", // Based on feedback from linguists quotes in Italian are not required to be the guillemets, even though CLDR says so
+        ].forEach(locale => this.skipLocales.add(locale));   
```

Comment:
Where is this.skipLocales used?

Reply:
https://github.com/iLib-js/i18nlint/blob/main/src/rules/ResourceRule.js#L90